nuxt.config.ts

```diff
@@ -8,7 +7,7 @@ export default {
    ** Headers of the page
    */
   head: {
-    title: pkg.name, // TODO change page title
+    title: '%s - PROJECT_NAME', // TODO change PROJECT_NAME
```

Comment:
what does the `%s` do?

Reply:
It takes the title from a page and pops it into the site title. Something like "Home - Payment Manager Admin" as opposed to it just being "Payment Manager Admin" across the entire site.